sklearn/model_selection/_validation.py

```diff
@@ -154,6 +155,12 @@ def cross_validate(estimator, X, y=None, *, groups=None, scoring=None, cv=None,
 
         .. versionadded:: 0.20
 
+    return_predictions : {'predict_proba', 'predict'}, default=None
+        Return cross-validation predictions for the dataset. 'predict' returns
+        the predictions whereas 'predict_proba' returns class probabilities.
```

Comment:
Maybe add what happens when None and add references to terms:
```suggestion
        What predictions to return. If 'predict', return output of :term:`predict` and if
        'predict_proba', return output of :term:`predict_proba`. The test set indices
        are also returned under a separate dict key.
        If None, do not return predictions or test data indices.
```

Reply:
I've included the suggested change (minus the reference test indices, for now).

I'm not entirely clear on when :term: is needed - seems like it's for string arguments. It would help if the example here showed and discussed using :term: : https://scikit-learn.org/dev/developers/contributing.html#documentation

Should I file a new issue, if there isn't one already?